CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+## [Unreleased]
```

Comment:
I noticed S3Proxy doesn't have a changelog so I thought I would add one so one can capture documenting changes as part of pull request.

Reply:
Please remove.  The release page duplicates this:

https://github.com/gaul/s3proxy/releases